addons/l10n_it_edi_doi/models/res_company.py

```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _name = 'res.company'
+    _inherit = 'res.company'
+
+    l10n_it_edi_doi_tax = fields.Many2one(
```

Comment:
Shouldn't it be stored (same for `l10n_it_edi_doi_fiscal_position`)?

Reply:
Removed the compute functions; stored the fields and put tax / fiscal position information
in file `template_it.py`.
That probably makes more sense.